src/components/list-blog/ListBlogComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import React, {useState, useEffect} from 'react' 
+import PaginationComponent from '../pagination/PaginationComponent';
+import PostComponent from '../post/PostsComponent';
+
+
+export default function ListBlogComponent() {
+    const [posts, setPosts] = useState([]);
+    const [currentPage, setCurrentpage] = useState(1);
+    const [postsPerPage, setPostsPerPage] = useState(10);
+
+
+    useEffect(() => {
+        const fetchPosts = async () => {
```

Comment:
You should make a folder called `src/services/blogServices.ts`, and define all of the services like this in that file.